src/main/java/org/broadinstitute/hellbender/tools/exome/gcbias/CorrectGCBias.java

```diff
@@ -0,0 +1,85 @@
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
+ */
+@CommandLineProgramProperties(
+        oneLineSummary = "Correct for per-sample GC bias effects.",
+        summary = "Correct for per-sample GC bias effects.",
+        programGroup = CopyNumberProgramGroup.class
+)
+public class CorrectGCBias extends CommandLineProgram {
+    public static final String INPUT_READ_COUNTS_FILE_LONG_NAME = StandardArgumentDefinitions.INPUT_LONG_NAME;
+    public static final String INPUT_READ_COUNTS_FILE_SHORT_NAME = StandardArgumentDefinitions.INPUT_SHORT_NAME;
+
+    public static final String OUTPUT_READ_COUNTS_FILE_LONG_NAME = StandardArgumentDefinitions.OUTPUT_LONG_NAME;
+    public static final String OUTPUT_READ_COUNTS_FILE_SHORT_NAME = StandardArgumentDefinitions.OUTPUT_SHORT_NAME;
+
+    @Argument(
+            doc = "Read counts input file.",
+            shortName = INPUT_READ_COUNTS_FILE_SHORT_NAME,
+            fullName = INPUT_READ_COUNTS_FILE_LONG_NAME,
+            optional = false
+    )
+    protected File inputReadCountsFile;
+
+    @Argument(
+            doc = "Read counts output file.",
+            shortName = OUTPUT_READ_COUNTS_FILE_SHORT_NAME,
+            fullName = OUTPUT_READ_COUNTS_FILE_LONG_NAME,
+            optional = false
+    )
+    protected File outputReadCountsFile;
+
+    // arguments for GC-annotated targets
+    @ArgumentCollection
+    protected TargetArgumentCollection targetArguments = new TargetArgumentCollection();
+
+    @Override
+    protected Object doWork() {
+        final TargetCollection<Target> targets = targetArguments.readTargetCollection(false);
+        final ReadCountCollection inputCounts = readInputCounts(targets);
+        final double[] gcContentByTarget = gcContentsOfTargets(inputCounts, targets);
+        final ReadCountCollection outputCounts = GCCorrector.correctCoverage(inputCounts, gcContentByTarget);
+        writeOutputCounts(outputCounts);
+        return "Success";
+    }
+
+    private ReadCountCollection readInputCounts(final TargetCollection<Target> targets) {
+        final ReadCountCollection inputCounts;
+        try {
+            inputCounts = ReadCountCollectionUtils.parse(inputReadCountsFile, targets, false);
+        } catch (final IOException ex) {
+            throw new UserException.CouldNotReadInputFile(inputReadCountsFile, ex.getMessage(), ex);
+        }
+        return inputCounts;
+    }
+
+    private void writeOutputCounts(final ReadCountCollection outputCounts) {
+        try {
+            ReadCountCollectionUtils.write(outputReadCountsFile, outputCounts);
+        } catch (final IOException ex) {
+            throw new UserException.CouldNotCreateOutputFile(outputReadCountsFile, ex);
+        }
+    }
+
+    private double[] gcContentsOfTargets(final ReadCountCollection inputCounts, final TargetCollection<Target> targets) {
```

Comment:
Can you file an issue to integrate it into the WDL workflow (make sure to tag it with wdl and milestone alpha-3)?

Also, coordinate with me to run this on somatic WGS and then somatic capture.